### PR TITLE
Update node to v16

### PR DESCRIPTION
**Pull request by aahlrichs5**
#4 
Node12 is deprecated. Update to use Node16

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ inputs:
     description: 'Output the Info.plist file in console'
     required: false
 runs:
-  using: 'node12'
+  using: 'node16'
```
Comment (HarelM) [inline action.yml:18]:
```suggestion
  using: 'node20'
```

**Comment by HarelM** [comment]
Can I use a fork for a github action? Will it use the main branch or can I use a different branch?
@aahlrichs5 what did you end up doing?

**Comment by HarelM** [comment]
@damienaicheh any updates on this?
I would like to resolve the warning I get in my build.
Let me know how I can help to push this forward.
image

**Comment by damienaicheh** [comment]
Hi, I didn't receive notification previously. Let me check this next week if I can

**Comment by HarelM** [comment]
Cool, thanks for responding so quickly!
If there's anything I can do to help, even testing a new version please let me know.

**Comment by damienaicheh** [comment]
I upgrade to Node 20. Thanks for your help.

**Pull request closed without merging**